app/src/main/java/org/fossasia/openevent/general/notification/Notification.kt

```diff
@@ -16,10 +17,10 @@ data class Notification(
     @Id(IntegerIdHandler::class)
     @NonNull
     @PrimaryKey
-    val id: Long,
+    val id: Int,
     val message: String? = null,
     val receivedAt: String? = null,
-    val isRead: Boolean? = null,
+    @JsonRawValue var isRead: Boolean = false,
```

Comment:
Incorrect solution

Reply:
Please suggest the solution because `KebabCaseStrategy` is skipping the `is` word

Reply:
@JsonProperty

Reply:
> @JsonProperty

I have tried it. Not working - 
```java
@JsonProperty("is-read")
    var isRead: Boolean = false,
```
Still sending `read`

Reply:
@get:JsonProperty("is_something")
@param:JsonProperty("is_something")

Reply:
param not working, `@get:JsonProperty` is working fine, Updated :+1: